src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3667,6 +3767,28 @@ private static ImmutableArray<Symbol> FixPartialMember(ImmutableArray<Symbol> sy
             return Remove(symbols, implementation);
         }
 
+        // PROTOTYPE(partial-properties): is there some abstraction that would make this nice?
```

Comment:
I can't think of a way to factor this further either. Consider dropping PROTOTYPE comment